### PR TITLE
Added pseudocode formulas for interest-is-interesting

### PR DESCRIPTION
**Pull request by honix**
I did struggle myself without formulas, and you can't find any clues how this will be done. I think this might help someone.

### CODE REVIEW

**Review by ErikSchierboom — COMMENTED** [review]
I agree that explaining the formulas can help. I'd prefer these formula hints to be added to the task-specific hints. You also mention that it is pseudocode, but you can almost copy-paste the code. I'm not sure what to do about that though.

**Comment by honix** [comment]
> You also mention that it is pseudocode, but you can almost copy-paste the code. I'm not sure what to do about that though.

What can I do - fsharp have so nice syntax, it's looks like pseudocode!

It will not work, actually. The task is forcing students to learn numeric types and type casting, so while there is no type casting in pseudocode, copy paste will now solve entire task.

**Comment by ErikSchierboom** [comment]
>  copy paste will now solve entire task.

You mean "will not", right? And I appreciate that F# is like pseudocode, so I'm fine with using it in the hints, but linked to their corresponding task :)

**Comment by honix** [comment]
> linked to their corresponding task 

Done!

File: exercises/concept/interest-is-interesting/.docs/hints.md [hunk 1]
```diff
@@ -8,18 +8,18 @@
 
 - When calculating interest, it might be helpful to convert a negative balance to a positive one. One could use arithmetic here, or [this method](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core-operators.html#abs).
 
+- **interest** balance = interestRate(balance) / 100 * balance
+
 ## 3. Calculate the annual balance update
 
 - When calculating the annual yield, it might be useful to temporarily convert a negative balance to a positive one. One could use arithmetic, or one of the built-in [math functions][math-functions].
 
+- **annualBalanceUpdate** balance = balance + interest(balance)
```
Comment (ErikSchierboom) [inline exercises/concept/interest-is-interesting/.docs/hints.md:17]:
```suggestion
- The formula to calculate the annual balance update is: `balance + interest of balance`
```

File: exercises/concept/interest-is-interesting/.docs/hints.md [hunk 2]
```diff
@@ -8,18 +8,18 @@
 
 - When calculating interest, it might be helpful to convert a negative balance to a positive one. One could use arithmetic here, or [this method](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core-operators.html#abs).
 
+- **interest** balance = interestRate(balance) / 100 * balance
```
Comment (ErikSchierboom) [inline exercises/concept/interest-is-interesting/.docs/hints.md:11]:
```suggestion
- The formula to calculate the interest is: `(interest rate of balance / 100) * balance`
```

File: exercises/concept/interest-is-interesting/.docs/hints.md [hunk 3]
```diff
@@ -8,18 +8,18 @@
...
 [math-functions]: https://www.dotnetperls.com/math-fs
 
-## 4. Formulas
+## 4. Calculate the amount to donate
 
-Note: this is pseudocode, not actual fsharp solution code.
-
-- **interest** balance = interestRate(balance) / 100 * balance
-- **annualBalanceUpdate** balance = balance + interest(balance)
 - **amountToDonate** balance _(when balance > 0)_ = balance * (taxFreePercentage / 100 * 2)
```
Comment (ErikSchierboom) [inline exercises/concept/interest-is-interesting/.docs/hints.md:24]: I can't do a proper suggestion, but maybe do something like:

```
- The formula to calculate the amount to donate is:
  - If the balance is <= 0: `0`
  - If the balance is > 0: `balance * (tax free percentage / 100 * 2)`
```

**Review by ErikSchierboom — COMMENTED** [review]
I've made some minor suggestion that still keep the formula intact, but don't allow for copy-pasting. What do you think?

**Comment by honix** [comment]
Yeah, I will apply these in a minute.

**Comment by honix** [comment]
Done.

**Review by ErikSchierboom — APPROVED** [review]
Lovely! Thanks for this!

**Pull request merged**